src/api/app/models/project.rb

```diff
@@ -350,7 +350,7 @@ def self.get_maintenance_project(at = nil)
     at ||= AttribType.find_by_namespace_and_name!('OBS', 'MaintenanceProject')
     maintenanceProject = Project.find_by_attribute_type(at).first
     unless maintenanceProject and check_access?(maintenanceProject)
-      raise UnknownProject.new 'There is no project flagged as maintenance project on server and no target in request defined.'
+      raise BsRequestAction::UnknownProject.new 'There is no project flagged as maintenance project on server and no target in request defined.'
```

Comment:
raising an action sounds wrong to me, project model should IMHO stay as basic as possible. You could get a request related exception when no request exists for this project.